projects/igniteui-angular/src/lib/directives/tooltip/tooltip.directive.ts

```diff
@@ -195,6 +198,12 @@ export class IgxTooltipTargetDirective extends IgxToggleActionDirective implemen
 
         this._overlayDefaults.positionStrategy = new AutoPositionStrategy(positionSettings);
         this._overlayDefaults.closeOnOutsideClick = false;
+        this.target.onClosed.subscribe(() => {
+            this.closeAnimationInProgress = false;
+        });
+        this.target.onOpened.subscribe(() => {
```

Comment:
I believe we should unsubscribe from the tooltip's onOpened event when the respective targets gets destroyed.

Reply:
done